docker/network/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:12
+
+RUN mkdir -p /usr/app/src \
+	&& mkdir -p /usr/app/media \
+	&& mkdir -p /usr/app/scripts
+
+COPY ./docker/network/bootstrap.sh /usr/app/scripts/bootstrap.sh
+COPY ./ /usr/app/src
+
+RUN chown -R node:node /usr/app
+
+WORKDIR /usr/app/src
+
+USER node
+
+ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
+ENV PATH="/home/node/.npm-global/bin:${PATH}"
+
+RUN npm install -g --loglevel warn nodemon bunyan && npm cache clean --force
+RUN npm install --loglevel warn
```

Comment:
check out this thread: https://myaegee.slack.com/archives/C45Q9DTMK/p1603991345073800, maybe we can make it nice from the start

Reply:
```
FROM node:12

RUN mkdir -p /usr/app/src \
	&& mkdir -p /usr/app/media \
	&& mkdir -p /usr/app/scripts

COPY --chown=node:node package.json /usr/app/src

ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
ENV PATH="/home/node/.npm-global/bin:${PATH}"

RUN npm install -g --loglevel warn nodemon bunyan && npm cache clean --force
RUN npm install --loglevel warn

COPY . /usr/app/src

RUN chown -R node:node /usr/app

USER node

COPY ./docker/network/bootstrap.sh /usr/app/scripts/bootstrap.sh

CMD sh /usr/app/scripts/bootstrap.sh && npm start

EXPOSE 8084
```

Like this?

Reply:
@linuxbandit you might also be interested in this

Reply:
this should do it I think

Reply:
yes, I'll add some touches later as this is merged already but it is good overall

Reply:
Sounds good!